src/mlpack/core/data/load_impl.hpp

```diff
@@ -290,7 +297,9 @@ bool Load(const std::string& filename,
   // Now transpose the matrix, if necessary.
   if (transpose)
   {
-    inplace_transpose(matrix);
+    bool is_transposed = inplace_transpose(matrix, fatal);
```

Comment:
```suggestion
    bool isTransposed = inplace_transpose(matrix, fatal);
```
Simple style guide fix; we should name variables in camelcase. :)